README.qmd

```diff
@@ -12,23 +12,19 @@ knitr:
 [![R-CMD-check](https://github.com/Robinlovelace/spanish_od_data/actions/workflows/R-CMD-check.yaml/badge.svg)](https://github.com/Robinlovelace/spanish_od_data/actions/workflows/R-CMD-check.yaml)
 <!-- badges: end -->
 
-This repo demonstrates how to download and use OD data from Spain, published by [transportes.gob.es](https://www.transportes.gob.es/ministerio/proyectos-singulares/estudios-de-movilidad-con-big-data/opendata-movilidad)
+**spanishoddata** provides functions for downloading and formatting Spanish origin-destination (OD) data from the Ministry of Transport and Sustainable Mobility of Spain.
 
-The data is provided as follows:
+The Spanish OD data is primarily sourced from mobile phone positioning data and includes matrices for overnight stays, individual movements, and trips of Spanish residents at different geographical levels.
```

Comment:
```suggestion
The OD data is from mobile phone positioning data and includes information movements over time (with hourly temporal resolution, allowing estimation of overnight stays for example) and broken down by demographic group and other variables between different geographic zones.
```

Thoughts also welcome here @e-kotov who has used the data loads.

Reply:
I meant that the data includes information in three types of matrices: 1) matrices with the number of overnight stays; 2) the number of people; and 3) the number of trips at different geographic levels.

Reply:
Oh yeah good point! That's now what is on main branch after merging this, we can always make further tweaks later.